extra_tests/snippets/syntax_non_utf8.py

```diff
@@ -5,8 +5,7 @@
 
 dir_path = os.path.dirname(os.path.realpath(__file__))
 
-# TODO: RUSTPYTHON, RustPython raises a SyntaxError here, but cpython raise a ValueError
-error = SyntaxError if platform.python_implementation() == 'RustPython' else ValueError
+error = ValueError
 with assert_raises(error):
```

Comment:
```suggestion
with assert_raises(ValueError):
```

Reply:
I applied your suggestion in [7e88c8f](https://github.com/RustPython/RustPython/pull/4052/commits/7e88c8ff55017f0c11b807d14e97398a650aaad5). Thank you! 🙏🏻